apps/web/src/components/layout/components/LocalStudioSidebar/LocalStudioSidebarToggleButton.tsx

```diff
@@ -8,7 +8,7 @@ import {
   getStudioWorkflowStepLink,
   getStudioWorkflowTestLink,
 } from '../../../../studio/utils/routing';
-import { NavMenuLinkButton, NavMenuToggleButton } from '../../../nav/NavMenuButton';
+import { NavMenuLinkButton, NavMenuToggleButton } from '../../../nav/NavMenuButton/index';
```

Comment:
Is there a specific reason we needed to add this import from `index`? In my experience, it's usually a symptom of a circular dependency in the tree.

Reply:
```suggestion
import { NavMenuLinkButton, NavMenuToggleButton } from '../../../nav/NavMenuButton';
```